pmd-core/src/main/java/net/sourceforge/pmd/cpd/CPDConfiguration.java

```diff
@@ -144,7 +142,11 @@ public class CPDConfiguration extends AbstractConfiguration {
     @Parameter(names = { "--debug", "--verbose", "-v", "-D" }, description = "Debug mode.")
     private boolean debug = false;
 
-    // this has to be a public static class, so that JCommander can use it!
+    /**
+     * @deprecated this is internal API. The class has to be public static, so that JCommander can use it.
+     */
+    @Deprecated
+    @InternalApi
```

Comment:
this doesn't even need to be marked as `@InternalApi`, we are not moving it to an `internal` package in PMD 7, but removing it altogether.

Reply:
done